packages/rule-no-p3p/README.md

```diff
@@ -0,0 +1,25 @@
+# no-p3p (`no-p3p`)
+
+Don't use P3P related headers
+
+## Why is this important?
+
+P3P -- Platform for Privacy Preferences Project -- is an outdated technology
+that is meant to allow browsers to programmatically check privacy policies. In
+case it's not kept in check with the actual privacy policies, it's a legal risk.
+
+## What does the rule check?
+
+It disallows the "P3P" header.
+
+### Examples that **trigger** the rule
+
+Any resource that has a "P3P" header. Contents of the header are not checked.
+
+### Examples that **pass** the rule
+
+Any resource that doesn't have a "P3P" header.
```

Comment:
In the same spirit as the previous one, an example with header response that doesn't have the P3P header.